src/Permanent/MediumPermanentPlainTextFormatter.php

```diff
@@ -83,10 +83,18 @@ private function generateWeekScheme(array $openingHoursData): string
 
         $weekScheme = $this->getWeekScheme($weekDaysOpen, $this->formatter);
 
+        $isFirstPeriodASingleDay = array_pop($weekScheme);
+
         // Put all the day names with opening hours on a single line with 'Open at' (sec) at the beginning.
         // E.g. 'Open at mo - th & su'
+        if ($isFirstPeriodASingleDay) {
```

Comment:
The comment above probably needs to be updated (it's example does not start with a single day)